### ISSUE TITLE
Missing BEGIN button

### ISSUE THREAD

**Issue by hrondra** [google/google-authenticator-android#73]
Cannot "begin" after installation - the application is in Czech on Sony E2105 with Android 4.4.4. I can see "Your password" and "A code this app will generate for you" (in my language Vaše heslo etc.) - but there is no button - it is probably "bellow the visible area of the screen"

**Comment by ThomasHabets** [comment]
Can you take a screenshot?

**Comment by TikhonovAlexander** [comment]
I also don't see BEGIN button on my device. 

Android: 7.0
App version: 5.00

Can someone help with this issue ?

![image](https://user-images.githubusercontent.com/7353054/35192965-0a431c62-feb5-11e7-9368-69f0d60ddf59.png)

**Comment by ThomasHabets** [comment]
Does it scroll? What's in the tripple-dot menu?

**Comment by TikhonovAlexander** [comment]
It doesn't scroll, in the menu - how it works, settings, support.

**Comment by ThomasHabets** [comment]
Hmm, that doesn't look good. Could you try switching your phone to English, set up your first account, and then switch it back?

(this just to confirm that this is a localisation issue, not that it's the permanent fix)

**Comment by ThomasHabets** [comment]
I've filed internal bug b/72278863 about this.

**Comment by TikhonovAlexander** [comment]
Yes, I have BEGIN button on English locale, thanks for quick fix!

**Comment by sjfoley** [comment]
I have the same problem when using larger fonts with an English locale.  The "Begin" button is missing, the screen is not scrollable, and there are no options to add an account from the menu.  Samsung S7, Android 8.0.0, Authenticator 5.0.0.
